backend/config/config.go

```diff
@@ -102,6 +102,9 @@ type Backend struct {
 
 	// BtcUnit is the unit used to represent Bitcoin amounts. See `coin.BtcUnit` for details.
 	BtcUnit coin.BtcUnit `json:"btcUnit"`
+
+	// RestartInTestnet represents whether the app should launch in testnet on the next start.
+	RestartInTestnet bool `json:"restartInTestnet"`
```

Comment:
Maybe a better name is `StartInTestnet` or just `Testnet`? The flag is used to turn into testnet when starting, not before. This way it sounds like it would trigger a restart or something like that.